appveyor.yml

```diff
@@ -1,3 +1,19 @@
+#
+# Copyright (C) 2016 Red Hat, Inc.
```

Comment:
Since I suppose everyone uses some sort of automatism from its IDE to create such headers automatically when creating new files, i meant it as a heads up to update this personal IDE config.

nothing more :)